### PR TITLE
Init `web3-eth-tx-utils` Base

### PR DESCRIPTION
**Pull request by Spacesai1or**
Creates the base for the `web3-eth-tx-utils` package

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -794,3 +798,4 @@ should use 4.0.1-alpha.0 for testing.
 #### web3-eth
 
 -   Moved the errors' classes from `web3-eth/src/errors.ts` to `web3-errors/src/errors/transaction_errors.ts` (#5462)
+-   Moved transaction related utils from `web3-eth` to the new package `web3-eth-tx-utils` (#5545)
```
Comment (spacesailor24) [inline CHANGELOG.md:801]:
```suggestion
```

Comment (Spacesai1or) [inline CHANGELOG.md]:
```suggestion
```

**Review by avkos — APPROVED** [review]

File: packages/web3-eth-tx-utils/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- EXAMPLE
+
+## [1.0.0]
+
+### Added
```
Comment (Muhammad-Altabba) [inline packages/web3-eth-tx-utils/CHANGELOG.md:12]: Could we please have a list of the features that have been added to this initial version of the package?
Thanks,

Comment (Muhammad-Altabba) [reply packages/web3-eth-tx-utils/CHANGELOG.md:12]: The package is totally empty :)

**Review by Muhammad-Altabba — APPROVED** [review]

**Comment by Muhammad-Altabba** [comment]
Do we really need to have this separate package? I mean, why to move the functionality from the sub-package: web3-eth into what seems to me as a sub-sub-package?

**Comment by Spacesai1or** [comment]
> Do we really need to have this separate package? I mean, why to move the functionality from the sub-package: web3-eth into what seems to me as a sub-sub-package?

@Muhammad-Altabba We've discussed this on calls before, but this is necessary to avoid  circular dependency issues for `web3-test-utils`. Additionally there was a user whom request access to some of the tx utils methods that we currently aren't exporting and didn't require the rest of `Web3Eth` (i.e. RPC method wrappers)

**Comment by jdevcs** [comment]
> > Do we really need to have this separate package? I mean, why to move the functionality from the sub-package: web3-eth into what seems to me as a sub-sub-package?
> 
> @Muhammad-Altabba We've discussed this on calls before, but this is necessary to avoid circular dependency issues for `web3-test-utils`. Additionally there was a user whom request access to some of the tx utils methods that we currently aren't exporting and didn't require the rest of `Web3Eth` (i.e. RPC method wrappers)

If one testing method is not applicable due to some constraints, instead of (structural refactors) changing product to be tested, best approach is to change testing method. @spacesailor24

**Pull request merged**